README.md

```diff
@@ -739,12 +771,12 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   ```swift
   // WRONG
-  if let galaxy = galaxy,
+  if let galaxy,
```

Comment:
Did you use a regex to search for other code examples to correct, or did you look through all of them?

Reply:
I think I looked at all cases of `if let` and `guard let` in the document